crates/biome_js_analyze/src/lint/style/use_self_closing_elements.rs

```diff
@@ -53,6 +57,31 @@ declare_lint_rule! {
     /// ```js
     /// <Foo.bar>child</Foo.bar>
     ///```
+    ///
+    /// ## Options
+    ///
+    /// #### `checkHtmlElements`
+    ///
+    /// Default: true
+    ///
+    /// This option allows you to specify whether or not to check native HTML elements.
+    ///
+    /// In the following example, when the option is set to "false", it will not self close native HTML elements.
```

Comment:
A suggestion because the following example is set to true.
```suggestion
    /// In the following example, when the option is set to "true", it will check for self-closing native HTML elements.
```

Reply:
Thanks for the suggestion, sorry the example should be set to `false` since `true` is the default behaviour. I’ll update it!

Reply:
Addressed in last commit, thank you!